sdk/python/kfp/_client.py

```diff
@@ -125,15 +125,19 @@ def __init__(self, host=None, client_id=None, namespace='kubeflow', other_client
     config = self._load_config(host, client_id, namespace, other_client_id, other_client_secret, existing_token, proxy, ssl_ca_cert)
     # Save the loaded API client configuration, as a reference if update is
     # needed.
+    self._load_context_setting_or_default()
     self._existing_config = config
-    api_client = kfp_server_api.api_client.ApiClient(config, cookie=cookies)
+    if cookies is None:
+      cookies = self._context_setting.get('cookie')
```

Comment:
Can you please rename all setting names to something easier to understand in broad context. For example `client_authentication_cookie`, `client_authentication_header_name`, `client_authentication_header_value`

Reply:
@Ark-kun done!